tests/myst/mkcpio/Makefile

```diff
@@ -16,7 +16,7 @@ REDEFINE_TESTS=1
 include $(TOP)/rules.mak
 
 tests:
-	$(RUNTEST) $(MAKE) test-mkcpio
```

Comment:
Is this intentional?

Reply:
yep! There are several things in the individual test that take a long time and each has $(runtest). Doing it at this layer can cause the test to timeout prematurely. Creating large cpio/ext2 images can take a bit of time.